code/modules/mob/living/simple_animal/hostile/deer.dm

```diff
@@ -51,11 +51,11 @@ Have them evaluate mobs in their vicinity
 
 		if(H.m_intent == "run") //Don't run while stalking deer
 			spook_prob = 70
-		if(istype (H:wear_suit, /obj/item/clothing/suit/leather/deer))
+		if(H.get_item_by_slot(slot_wear_suit) == /obj/item/clothing/suit/leather/deer)
```

Comment:
Right. Why didn't you tell me this in the first place rather than just

![holyfuck](https://cloud.githubusercontent.com/assets/9110500/21487754/2747e424-cbca-11e6-8650-3771bd4c5516.png) ?

Could save everybody a lot of time if you stipulated specifically what needed to be changed rather than throwing process names at people.